DataFormats/HcalRecHit/interface/HBHERecHit.h

```diff
@@ -54,6 +57,7 @@ class HBHERecHit : public CaloRecHit {
   float chiSquared_;
   float rawEnergy_;
   float auxEnergy_;
+  float addEnergy_;
```

Comment:
what is the logic for this name "add"?

Reply:
"add" stay for additional.

The final idea is to have the M0 (no PU mitigation) and Mahi (in replacement of both M2 and M3).
This additional energy is mainly for cross testing M3 vs M2 vs Mahi.
Let's discuss how we want to do this at the next RECO meeting (17th november)

Reply:
auxiliary also means additional.
So, instead of expanding by synonyms, maybe aux1Energy (just the first index) or aux2Energy (to imply method2).
Perhaps it's time to give a bit more substance to the name to know what's inside.

If this is needed for comparisons, is it obvious that just energy is enough and you wouldn't need chi2, time, or some other flags?

Still, if this is needed for one pre-release, the addition of extra parameter(s) is rather wasteful. It seems like it is possible to do much more for comparisons by having another collection in parallel.